---
title: "Exploring impact of recombination landscapes on breeding outcomes"
collection: publications
category: manuscripts
permalink: /publication/2009-10-01-paper-title-number-1
excerpt: 'Simulated the effects of mutant recombination landscapes in maize and rice breeding.'
date: 2023-03-27
venue: 'Proceedings of the National Academy of Sciences.'
paperurl: 'https://doi.org/10.1073/pnas.2205785119'
citation: 'Nikita Sajai, Co-author. (2023). &quot;Exploring impact of recombination landscapes on breeding outcomes.&quot; <i>Proceedings of the National Academy of Sciences</i>. 120 (14).'
---
Plant breeding relies on crossing-over to create novel combinations of alleles needed to confer increased productivity and other desired traits in new varieties. However, crossover (CO) events are rare, as usually only one or two of them occur per chromosome in each generation. In addition, COs are not distributed evenly along chromosomes. In plants with large genomes, which includes most crops, COs are predominantly formed close to chromosome ends, and there are few COs in the large chromosome swaths around centromeres. This situation has created interest in engineering CO landscape to improve breeding efficiency. Methods have been developed to boost COs globally by altering expression of anti-recombination genes and increase CO rates in certain chromosome parts by changing DNA methylation patterns. In addition, progress is being made to devise methods to target COs to specific chromosome sites.

We review these approaches and examine using simulations whether they indeed have the capacity to improve efficiency of breeding programs. We found that the current methods to alter CO landscape can produce enough benefits for breeding programs to be attractive. They can increase genetic gain in recurrent selection and significantly decrease linkage drag around donor loci in schemes to introgress a trait from unimproved germplasm to an elite line. Methods to target COs to specific genome sites were also found to provide advantage when introgressing a chromosome segment harboring a desirable quantitative trait loci. We recommend avenues for future research to facilitate implementation of these methods in breeding programs.

**Tools:**
* AlphaSimR in R for breeding simulations
* Unix/Linux shell scripting using bedtools and samtools for genomic data processing
* git version control
